util/stringutil/string_util.go

```diff
@@ -239,3 +240,10 @@ func DoMatch(str string, patChars, patTypes []byte) bool {
 	}
 	return sIdx == len(str)
 }
+
+var reBlank = regexp.MustCompile(`\s`)
+
+// RemoveBlanks remove all blanks, returns a new string.
+func RemoveBlanks(s string) string {
```

Comment:
Don't forget unit test when you add something.
Personally speaking, I dislike regexp for it's poor performance.